src/search/models.py

```diff
@@ -73,6 +73,10 @@ class SearchPage(models.Model):
         _('Additional page content'),
         blank=True,
         help_text=_('Hidden content, revealed with a `See more` button'))
+    tab_title = models.CharField(
+        'Titre de l\'onglet principal',
```

Comment:
```suggestion
        "Titre de l'onglet principal",
```
Pardon je peux pas m'empêcher :)
(en vrai c'est plus lisible je trouve, et on retrouve plus facilement si on fait une recherche dans le code !)